prisma/schema.prisma

```diff
@@ -31,6 +31,8 @@ model User {
   companyId    String?
   created      DateTime      @default(now())
   updated      DateTime      @updatedAt
+  passwordResetToken String?
+  passWordResetTokenExpiry DateTime ? 
```

Comment:
Another way to do this would be to use a JWT with an expiration date, meaning we only store the token in the database and once it expires its invalid until a user asks to reset again.